### PR TITLE
Make checkout ssh key req more explicit

### LINKED ISSUE #2594: Rerun build with SSH: No SSH key is found
# Missing Information
When running a build with SSH.  I've gotten the following error:
```
Failed to enable SSH
No SSH key is found. Please make sure you’ve added at least one SSH key in your VCS account.
```
and the documentation on https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci2/ssh-access-jobs.md does not address how to fix this.

It's quite confusing because there seems to be many different ways to add SSH keys (Checkout SSH Keys? SSH Permissions? `add_ssh_keys` ?)

# Already Looked
https://circleci.com/docs/2.0/add-ssh-key/
https://discuss.circleci.com/t/unable-to-build-with-ssh/22372/3

It's still unclear what SSH key is necessary for this to work...

# Potential Locations
https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci2/ssh-access-jobs.md

### PR DESCRIPTION
**Pull request by smart-alek**
# Description
This highlights the need for adding an SSH key with "Checkout SSH Key" for either

# Reasons
Fixes #2594 and resolves [this JIRA issue](https://circleci.atlassian.net/browse/CIRCLE-13159).

### CODE REVIEW

**Comment by timc13** [comment]
I think it would be helpful to be even more clear about this:

[This page](https://circleci.com/docs/2.0/gh-bb-integration/#enable-your-project-to-check-out-additional-private-repositories) should state that a user key is necessary for debugging with SSH.  It currently only states that a user key is necessary for write access to your repository or for checking out additional repositories.  To be totally transparent, you should explain why a write access key is even necessary for SSH debugging.

File: jekyll/_cci2/ssh-access-jobs.md [hunk 1]
```diff
@@ -18,19 +18,23 @@ This document describes how to access a build container using SSH on CircleCI 2.
...
 things like log files, running processes, and directory paths. CircleCI 2.0 gives you the option to access all jobs via SSH.
 
-When you log in with SSH, you are running an interactive login shell. You are also likely to be running the command on top of the directory where the command failed the first time, so you are not starting a clean run. In contrast, CircelCI uses a non-interactive shell for running commands by default. Hence, steps run in interactive mode may succeed, while failing in non-interactive mode. 
+When you log in with SSH, you are running an interactive login shell. You are also likely to be running the command on top of the directory where the command failed the first time, so you are not starting a clean run. In contrast, CircelCI uses a non-interactive shell for running commands by default. Hence, steps run in interactive mode may succeed, while failing in non-interactive mode.
 
-1. To start a job with SSH enabled, select the 'Rebuild with SSH' option from
+## Steps
+
+1. Ensure that you have already [added an SSH key]({{ site.baseurl }}/2.0/gh-bb-integration/#enable-your-project-to-check-out-additional-private-repositories) for either GitHub or Bitbucket.
```
Comment (felicianotech) [inline jekyll/_cci2/ssh-access-jobs.md]: This isn't a step for debugging a build via SSH. Only if you need the container to connect via SSH to somewhere else.

File: jekyll/_cci2/ssh-access-jobs.md [hunk 2]
```diff
@@ -18,19 +18,23 @@ This document describes how to access a build container using SSH on CircleCI 2.
 Often the best way to troubleshoot problems is to SSH into a build container and inspect 
 things like log files, running processes, and directory paths. CircleCI 2.0 gives you the option to access all jobs via SSH.
 
-When you log in with SSH, you are running an interactive login shell. You are also likely to be running the command on top of the directory where the command failed the first time, so you are not starting a clean run. In contrast, CircelCI uses a non-interactive shell for running commands by default. Hence, steps run in interactive mode may succeed, while failing in non-interactive mode. 
+When you log in with SSH, you are running an interactive login shell. You are also likely to be running the command on top of the directory where the command failed the first time, so you are not starting a clean run. In contrast, CircelCI uses a non-interactive shell for running commands by default. Hence, steps run in interactive mode may succeed, while failing in non-interactive mode.
```
Comment (felicianotech) [inline jekyll/_cci2/ssh-access-jobs.md]: I don't think this was updated in this PR but CircleCI was spelled as `CircelCI` here.

File: jekyll/_cci2/ssh-access-jobs.md [hunk 3]
```diff
@@ -18,19 +18,23 @@ This document describes how to access a build container using SSH on CircleCI 2.
 Often the best way to troubleshoot problems is to SSH into a build container and inspect
```
Comment (felicianotech) [inline jekyll/_cci2/ssh-access-jobs.md]: Maybe "is to SSH into a build container" -> "is to SSH into a Job"

My reasoning, a Job can run in a container for the `docker` executor but a VM for the `machine` or `macos` executors. It's not always containers.

**Comment by smart-alek** [comment]
@timc13 looks like I overthought this one! 😬 As long as you've added an SSH key to your VCS account, we'll do the rest and you should be able to SSH into the job to see what's happening.

But! Are you still stuck? Have you been able to SSH into a job yet?

**Comment by timc13** [comment]
my comments still stand.  I have been able to SSH into a job, but that's after much trial and error and not information learned from the documentation.

**Pull request merged**